code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -183,13 +188,23 @@
 		M.Dizzy(5)
 		if(iscultist(M) && prob(5))
 			M.say(pick("Av'te Nar'sie","Pa'lid Mors","INO INO ORA ANA","SAT ANA!","Daim'niodeis Arc'iai Le'eones","R'ge Na'sie","Diabo us Vo'iscum","Eld' Mon Nobis"))
+		else if(is_servant_of_ratvar(M) && prob(5))
+			switch(pick("speech", "message", "emote"))
+				if("speech")
+					M.say("...[pick("Ratvar... lbhe yvtug tebjf qnex", "Jurer ner lbh, znfgre?", "Ur yvrf ehfgvat va Reebe", "Chetr nyy hagehguf naq... naq... fbzrguvat")]...")
+				if("message")
+					M << "<span class='warning'><b>[pick("Ratvar's illumination of your mind has begun to flicker.", "He lies rusting in Reebe, derelict and forgotten. And there he shall stay.", \
+					"You can't save him. Nothing can save him now.", "It seems that Nar-Sie will triumph after all.")]</b></span>"
```

Comment:
It makes perfect sense from the ic standpoint and perspective of a follower.
By denying ratvar his followers, you are preventing his release from the celestial derelict. The line of nothing can save him now, is the holy water eroding their faith in ratvar.
Since keeping ratvar locked up in reebe is part of narsie's plan, and ratvar is the only real one who can stand up to narsie (it is implied in the lore that narsian worship is spread across multiple systems, so stopping their rise on the station doesn't actually stop them forever). Therefore, narsie will rise and triumph.